scripts/deployment/goerli-testnets.json

```diff
@@ -0,0 +1,61 @@
+{
+  "L1": {
+    "name": "Goerli",
+    "rpc": "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
+    "chain_id": 5
+  },
+  "L2": [
+    {
+      "name": "Arbitrum Goerli",
+      "rpc": "https://goerli-rollup.arbitrum.io/rpc",
+      "chain_id": 421613,
+      "l1_messenger": [
+        "ArbitrumL1Messenger",
+        "0xaf4159A80B6Cc41ED517DB1c453d1Ef5C2e4dB72",
+        "0x6BEbC4925716945D46F0Ec336D5C2564F419682C"
+      ],
+      "l2_messenger": "ArbitrumL2Messenger",
+      "finality_period": 604800,
```

Comment:
Just to make sure: is this finality period correct?

Reply:
However, we need to take care that we don't get below the actual finality period

Reply:
By testing the L2 -> L1 messaging, i noticed waiting times between 30 - 60 minutes. 
Their ETH bridge L2 -> L1 takes ~ hours. 
L1 -> L2 is shorter, ~ 15-30 minutes.

So just to be safe i will use 5 hours here.

Reply:
✅